src/config.ts

```diff
@@ -24,8 +24,8 @@ export interface Routes {
 }
 
 export interface Deployment {
-  accountId: string
-  zoneId: string
+  accountId?: string
+  zoneId?: string
```

Comment:
These are unused, leaving them here as optional properties, one might find it helpful to identify deployments in their worker code....